tachyon/crypto/commitments/merkle_tree/field_merkle_tree/field_merkle_tree_mmcs.h

```diff
@@ -98,6 +98,18 @@ class FieldMerkleTreeMMCS final
     return true;
   }
 
+  [[nodiscard]] bool DoCommitOwned(
+      std::vector<math::RowMajorMatrix<F>>&& owned_matrices,
+      Commitment* commitment, ProverData* prover_data) const {
+    TRACE_EVENT("ProofGeneration", "FieldMerkleTreeMMCS::DoCommit");
```

Comment:
Can you change these to match the function name?
```suggestion
    TRACE_EVENT("ProofGeneration", "FieldMerkleTreeMMCS::DoCommitOwned");